AutoRest/Generators/Extensions/Extensions.Tests/MappingTests.cs

```diff
@@ -33,10 +33,9 @@ public void TestInputMapping()
             generator.Generate(clientModel).GetAwaiter().GetResult();
             string body = settings.FileSystem.ReadFileAsText("X:\\Payload.cs");
```

Comment:
I fixed the test that had been disabled to not have a hard coded path and everything is now passing @fearthecowboy can you take a quick look?